### ISSUE TITLE
Few issues that I'm having right now.

### ISSUE THREAD

**Issue by SolarXD** [gravitylow/AntiCheat#273]
First off thank you for the plugin. I've moved from NCP and so far seemed to be a bit easier to manage but I am running into a few issues. A good few of them are with essentials.

/speed flags users as hacking(walking or flying)
/god and walking over lily pads will give intermittent damage
/god while flying/falling will issue damage or freeze you in the air
/r (herochat or essentials) counts as command spam(would be nice to have an excluded commands)

EDIT: Players are also reporting that at anytime they lag it will kick them for fastplace

Thank you for your time.

Using build #564

**Comment by SolarXD** [comment]
I'm having to adjust everything to the point that I might as well not even use anticheat. Fastplace, sprinting, flying are the most common ones. I've had to adjust them to the point that hacked clients are somewhat successful. The most irritating part is it isn't all players. I've been able to replicate most of it other than the issue with falling with /god on. It just seems to be random players at random times. Yes it is possible some could be using hacked clients but it is unlikely that the majority of them are.
